test/ProjectService.Data.UnitTests/GetUserProjectsRepositoryTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Provider;
+using LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.ProjectService.Data.UnitTests
+{
+    public class GetUserProjectsRepositoryTests
```

Comment:
There are not enough tests that check that the repository does not ignore the "showNotActive" variable

Reply:
Added